enhancements/operator-conditions.md

```diff
@@ -118,6 +123,14 @@ The `Upgradeable` condition might be useful when:
 - An operator is about to start a critical process and should not be upgraded until after the process is completed.
 - The operator is performing a migration of CRs that must be completed before the operator is ready to be upgraded.
 
+#### Ready
+
+The `Ready` "OLM Supported Condition" allows an operator to communicate its readiness to OLM. By default, OLM will move a CSV in the `Succeeded` Phase to the `Failed` phase if the deployments [Readiness Probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) is not true. By setting the `Ready` Condition, OLM will:
```

Comment:
Is there something missing after the last line `By setting the Ready Condition, OLM will:`?

Reply:
Leftover copy-paste, I'll remove this.